src/components/AppComponent.tsx

```diff
@@ -209,16 +211,29 @@ const AppComponent: FC<Props> = props => {
     // mobile safari must be detected because empty and full bullet points in Helvetica Neue have different margins
     mobile: isTouch,
     android: isAndroid,
+    native: Capacitor.isNativePlatform(),
     'drag-in-progress': dragInProgress,
     chrome: /Chrome/.test(navigator.userAgent),
     safari: isSafari(),
   })
 
+  useEffect(() => {
+    if (Capacitor.isNativePlatform()) {
+      StatusBar.setStyle({ style: Style.Dark })
```

Comment:
You should set `Style.Dark` or `Style.Light` depending on the value of the `dark` prop.